src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,76 @@
-import React from 'react';
+import React, { useState, Dispatch, SetStateAction } from 'react';
+import classNames from 'classnames';
+import { client } from '../utils/fetchClient';
+import { Post } from '../types/Post';
+
+interface T {
+  selectedPost: Post;
+  setIsLoading: Dispatch<SetStateAction<boolean>>;
+  isLoading: boolean;
+}
+
+export const NewCommentForm: React.FC<T> = ({
+  selectedPost,
+  setIsLoading,
+  isLoading,
+}) => {
+  const [inputName, setInputName] = useState('');
+  const [inputEmail, setInputEmail] = useState('');
+  const [inputText, setInputText] = useState('');
+  const [inputNameSend, setInputNameSend] = useState(false);
+  const [inputEmailSend, setInputEmailSend] = useState(false);
+  const [inputTextSend, setInputTextSend] = useState(false);
```

Comment:
boolean values must start with an `is` prefix almost every time - https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig